src/CUDAdrv.jl

```diff
@@ -5,17 +5,15 @@ module CUDAdrv
 using Compat
 using Compat.String
 
-const ext = joinpath(dirname(@__DIR__), "deps", "ext.jl")
-if isfile(ext)
-    include(ext)
-elseif haskey(ENV, "ONLY_LOAD")
+include("../deps/ext.jl")
```

Comment:
Agreed, but Base seems to be moving in the other direction (eg. JuliaLang/julia#21405).